### PR TITLE
new production environment configuration

### PR DESCRIPTION
**Pull request by reedhillmar**
____ Wrote Tests x Implemented ____ Reviewed

Neccesary checkmarks:

- [x] All Tests are Passing
- [x] The code will run locally

Type of change

- [x] New feature
- [ ] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

- [ ] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything
- [x] I don't know yet

Testing Changes

- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

- [ ] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

TY

### CODE REVIEW

**Review by ConnorRichmond — APPROVED** [review]

File: config/database.yml [hunk 1]
```diff
@@ -79,8 +79,18 @@ test:
 # Read https://guides.rubyonrails.org/configuring.html#configuring-a-database
 # for a full overview on how database connection configuration can be specified.
 #
+# production:
+#   <<: *default
+#   database: board_game_geek_data_production
+#   username: board_game_geek_data
+#   password: <%= ENV["BOARD_GAME_GEEK_DATA_DATABASE_PASSWORD"] %>
+
 production:
```
Comment (lanebret85) [inline config/database.yml:88]: beautiful configuration! glad to see we didn't need the url key added in too. great work!

File: config/database.yml [hunk 2]
```diff
@@ -79,8 +79,18 @@ test:
...
-  database: board_game_geek_data_production
-  username: board_game_geek_data
-  password: <%= ENV["BOARD_GAME_GEEK_DATA_DATABASE_PASSWORD"] %>
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  database: board_game_data_aws
+  username: postgres
+  password: bgb2307!
+  host: board-game-data-aws.ch6oocyqeqsl.us-east-2.rds.amazonaws.com
```
Comment (lofi-nowhale) [inline config/database.yml:96]: hey do we need to include our db URL also ... theres an exposed password that we might want to encrypt! 

Wanna hop on a zoom all together and check this out?

**Review by lofi-nowhale — CHANGES_REQUESTED** [review]
URL and Password comments in the code! :)

File: boardgamebuddy.pem
Comment (lanebret85) [inline boardgamebuddy.pem:1]: probs not a great idea to expose this. refactor?

File: config/database.yml (hunk 2 above)
Comment (lanebret85) [reply config/database.yml:96]: Let's do it! Teamwork will make the dream work.

File: boardgamebuddy.pem
Comment (lofi-nowhale) [reply boardgamebuddy.pem:1]: maybe a good thing to add to the gitignore?

Comment (lanebret85) [reply boardgamebuddy.pem:1]: Good idea. Then we can share it offline so each of us has it locally too

File: config/database.yml (hunk 2 above)
Comment (reedhillmar) [reply config/database.yml:96]: Good call! I'll add it to the rails credentials

File: boardgamebuddy.pem
Comment (reedhillmar) [reply boardgamebuddy.pem:1]: I'll delete in this PR and add to gitignore!

**Review by ConnorRichmond — APPROVED** [review]
nice hiding the database info

**Review by lofi-nowhale — APPROVED** [review]
thanks for getting those things encrypted + removed:) 'preciate you!

**Pull request merged**